scout/server/blueprints/variants/views.py

```diff
@@ -299,7 +300,12 @@ def cancer_variants(institute_id, case_name):
         store, institute_id, case_name, variants_query, form, page=page
     )
 
-    return dict(variant_type=variant_type, cytobands=cytobands, **data)
+    return dict(
+        variant_type=variant_type,
+        cytobands=cytobands,
+        **data,
+        expand_search=str(request.method == "POST"),
```

Comment:
```suggestion       
        expand_search=str(request.method == "POST"),
        **data
```
Being consequent with the other returned dicts 🤓

Reply:
👍